# cd /data/openpilot && curl https://raw.githubusercontent.com/misterkoll/openpilot-patches/master/noctua_fan_speed_patch_0.6.6.diff | git apply -v
#
# This patch sets fan speeds to max (65535) for top three temp breakpoints, suitable for EONs with Noctua fan mod.
#
# Credit for the content of this patch goes to Erich Moraga (https://github.com/ErichMoraga/)
#
===================================================================
--- a/selfdrive/thermald.py
+++ b/selfdrive/thermald.py
@@ -83,8 +83,8 @@
 # temp thresholds to control fan speed - low hysteresis
 _TEMP_THRS_L = [42.5, 57.5, 72.5, 10000]
 # fan speed options
-_FAN_SPEEDS = [0, 16384, 32768, 65535]
+_FAN_SPEEDS = [0, 65535, 65535, 65535]
 # max fan speed only allowed if battery is hot
-_BAT_TEMP_THERSHOLD = 45.
+_BAT_TEMP_THERSHOLD = 30.
